tools/st2_deploy.sh

```diff
@@ -176,7 +176,7 @@ install_apt() {
   fi
 
   # Add StackStorm APT repo
-  echo "deb http://downloads.stackstorm.net/deb/ trusty_unstable main" > /etc/apt/sources.list.d/stackstorm.list
+  echo "deb https://downloads.stackstorm.net/deb/ trusty_unstable main" > /etc/apt/sources.list.d/stackstorm.list
```

Comment:
I think @DoriftoShoes explicitly reverted it back to http a while ago because of some compatibility issues.

Using https would be good, but it's a not a huge deal breaker here since the packages are already signed.